locales/tr.json

```diff
@@ -2,7 +2,8 @@
     "_language": {
         "code": "tr",
         "dir": "ltr",
-        "name": "Türkçe"
+        "name": "Türkçe",
+        "fallback": "tr"
```

Comment:
does it makes sense?

Reply:
smol poke @Axolotle , there's a similar thing for "id" / indonesion, sounds like something automatically added by Weblate somehow ?